test/extensions/filters/http/ext_authz/ext_authz_integration_test.cc

```diff
@@ -292,6 +292,107 @@ class ExtAuthzHttpIntegrationTest : public HttpIntegrationTest,
   )EOF";
 };
 
+class ExtAuthzDenyAtDisableIntegrationTest : public Grpc::GrpcClientIntegrationParamTest,
```

Comment:
Sorry, I missed this in my previous review. Do you think it can be part of `ExtAuthzGrpcIntegrationTest`? It seems like the setup is very similar.

Reply:
Merged, with some little refactoring